roles/DCOS.requirements/tasks/main.yml

```diff
@@ -23,6 +23,14 @@
     enabled: false
   when: "'dnsmasq.service' in ansible_facts.services"
 
+- name: Check if docker is already installed
+  yum:
+    list:
+      - docker
+      - docker-ce
+  # If not installed yum_list.results[*].yumstate != installed
```

Comment:
the comment should be removed at this point. Even better its meaning should be ported into a task.